### DISCUSSION TITLE
Can you hold .md pages within another folder ? How would you reference them?

### DISCUSSION THREAD

**Discussion by Firelord710** [squidfunk/mkdocs-material#7448] [Q&A]
Hoping to add a folder within my directory in order to organize my pages (I have a lot of them). 

When dropping them into a new dir, i get 404 errors.

When changing the docs_dir in the mkdocs.yml, I am then not able to access both directories holding pages. 

Is there a way to have multiple docs_dir ? 

Thank you !

**Comment by alexvoss** [comment]
When you say "a new dir", you don't mean a subdirectory not within `docs/` but in the project root? What would be the advantage you seek over having subdirectories in `docs/`? 

If you do want to have multiple directories at the project level, there are a number of candidate options, depending on exactly what you need. The [Insiders Edition](https://squidfunk.github.io/mkdocs-material/insiders/) contains the [projects plugin](https://squidfunk.github.io/mkdocs-material/plugins/projects/), which allows you to have multiple sub-projects that each have their own `docs/` and can have different configurations. 

There is also Backstage's [mkdocs-monorepo-plugin](https://github.com/backstage/mkdocs-monorepo-plugin, which provides functionality that seems similar, but I don't have experience with that.

**Reply by Firelord710** [reply]
@alexvoss I have approximately 3200 pages and growing, many of which fit within certain categories. I was hoping to organize them within docs, but found that doing so made them inaccessible.

**Reply by alexvoss** [reply]
The location within the docs directory is reflected in the URL. I am unaware of an option to change the arrangement of the files without affecting the URL but you can [overwrite the destination URL with a hook](https://github.com/squidfunk/mkdocs-material/discussions/4693).

**Reply by squidfunk** [reply]
Please provide more information, ideally a reproducible example. I'm not sure we understand exactly what you're trying to achieve 😅

**Reply by Firelord710** [reply]
I'm so sorry let me better explain:

My docs/ folder contains about 3200 individual .md files for different pages of the site. 
I was hoping to sort them into category folders to simplify workflow, ex:

docs/dataset1
docs/dataset2
docs/etcetc

However, when I put the pages into folders within the docs/ folder, they become inaccessible via Navigation or directly visiting the link. 
Am I missing something required to make these pages accessible ? 

Thank you guys again.

**Reply by squidfunk** [reply]
Yes, probably. However, without knowing what you explicit configured, we can't know where to guide you. It's a good idea to provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/), so we can establish a common ground.

Please do not include all 3,200 pages, make it as minimal as possible.

**Reply by squidfunk** [reply]
@Firelord710 you closed the issue – could you please elaborate what solved the problem? This will help other users in the future that run into similar challenges ☺️
